Sources/JWT/Claims/AudienceClaim.swift

```diff
@@ -12,13 +12,13 @@ public struct AudienceClaim: NodeFailableInitializable {
         self.value = strings
     }
 
-    public init?(_ node: Node) {
+    public init(node: Node) throws {
```

Comment:
This is a breaking change. We need to supply a deprecated extension for the old initializer.